src/PokemonGymImpl.java

```diff
@@ -251,13 +266,21 @@ public void attackOrChange(Pokemon pokemon, Pokemon gymPokemon, PokemonTrainer t
         String choice = speler_A.nextLine();
 
         if (choice.equalsIgnoreCase("a")) {
+            // Attack
             String attack = chooseAttackPlayer(pokemon);
             performAttackPlayer(pokemon, gymPokemon, attack);
-        } else {
+
+            //Toevoeging Bonusopdracht 3:
+        } else if (choice.equalsIgnoreCase("c")) {
+            // Change Pokemon
             pokemon = choosePokemon(trainer);
             attackOrChange(pokemon, gymPokemon, trainer, gym);
             fightRound(trainer, gym, pokemon, gymPokemon);
+        } else if (choice.equalsIgnoreCase("f")) {
```

Comment:
Je check hier wel voor de optie "f", maar die optie staat niet in de prompt die je zojuist voor de gebruiker geprint hebt. De gebruiker weet dus niet dat deze ook "f" kan kiezen.